HCIDOCS-651: Oracle Cloud C3 (#88897)
openshift/openshift-docs: modules/c3-assisted-installer-overview.adoc

// Module included in the following assemblies:
//
// * installing/installing_oci/installing-c3-assisted-installer.adoc

:_mod-docs-content-type: PROCEDURE
[id="c3-ai-overview_{context}"]
= Overview

You can install {product-title} on {oci-c3-no-rt} by using the {ai-full}. 

> If you want to set up the cluster manually or using other automation tools, or if you are working in a disconnected environment, you can use the Red Hat Agent-based Installer for the installation. For details, see 'Installing a cluster on {oci-c3} by using the Agent-based Installer'. *[Will replace with link when I have it. Copied the paragraph from OCI, is the first part of the first sentence still correct?]*

Author: Please note question here.

+
--
** The first script builds IAM Resources in the {oci} Home region of the {oci-c3-short} system (two Dynamic Groups and one Policy).

** The second script builds the infrastructure resources on the {oci-c3-short} system to support the RHOCP cluster (the RHOCP VCN,  public and private subnets, load balancers, Internet GW, NAT GW, and DNS server). It includes all the resources needed to activate the control plane nodes and compute nodes that form a cluster. 
--
+
The bastion host is installed in the designated RHOCP Compartment and configured to communicate through a designated {oci-c3-short} DRG Subnet or Internet GW Subnet within the {oci-c3-short} parent tenancy. 

* The second instance is the `jumpserver`. The jumpserver is configured as a web server to communicate to the RHOCP cluster. The jumpserver is installed with the {product-title} client CLI, within the {product-title} VCN built by the Terraform scripts.

The installation process subsequently provisions three control plane (master) nodes and three compute (worker) noodes, together with the external and internal Load Balancers that form the cluster. 

.Main steps

> The main steps of the procedure are as follows:

Author: Is the whole process day 1 or day 2? I haven't related to day 2 so maybe I need to add an explanation.

// Module included in the following assemblies:
//
// * installing/installing_oci/installing-c3-assisted-installer.adoc

:_mod-docs-content-type: CONCEPT
[id="c3-ai-overview_{context}"]

> = Overview

Reviewer: Per the [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings), we should not use just "overview" as a heading. Maybe something like "Overview of...." or "About installing...." would work here.

This section explains how to install {product-title} on {oci-c3-no-rt} by using the {ai-full}. 

For an alternative installation method, see 'Installing a cluster on {oci-c3} by using the Agent-based Installer'.

.Preinstallation considerations

* Ensure that your installation meets the prerequisites specified for Oracle. For details, see link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_assisted_installer.pdf?source=:em:nl:mt::::PCATP[Access and Considerations] in the Oracle documentation.

* Ensure that your infrastructure is certified and uses a compatible cloud instance type. For details, see link:https://catalog.redhat.com/cloud/detail/216977[Oracle Cloud Infrastructure].

* Ensure that you are performing the installation on a virtual machine.

.Installation process

> The installation process builds a `bastion`  host within the designated compartment of the {product-title} cluster. The `bastion` host is used to run two Terraform scripts:

Reviewer 2: ```suggestion
The installation process builds a `bastion` host within the designated compartment of the {product-title} cluster. The `bastion` host is used to run two Terraform scripts:
```
Is there a reason why "bastion" is in backticks here, but not in backticks later in the content?

* The first script builds IAM Resources in the {oci} Home region of the {oci-c3-short} system (two Dynamic Groups and one Policy).

* The second script builds the infrastructure resources on the {oci-c3-short} system to support the RHOCP cluster (the RHOCP VCN,  public and private subnets, load balancers, Internet GW, NAT GW, and DNS server). It includes all the resources needed to activate the control plane nodes and compute nodes that form a cluster. 

> The bastion host is installed in the designated RHOCP Compartment and configured to communicate through a designated {oci-c3-short} DRG Subnet or Internet GW Subnet within the {oci-c3-short} parent tenancy. 

Reviewer 2: ```suggestion
The bastion host is installed in the designated {product-title} Compartment and configured to communicate through a designated {oci-c3-short} DRG Subnet or Internet GW Subnet within the {oci-c3-short} parent tenancy. 
```

The installation process subsequently provisions three control plane (master) nodes and three compute (worker) nodes, together with the external and internal Load Balancers that form the cluster. This is the standard implementation for {oci-first-no-rt}. 

.Main steps

The main steps of the procedure are as follows:

. Installing the Terraform application and the {oci} command line interface (CLI) on an {oci-c3-short} bastion server.

> . Executing the Terraform script from the application (part 1) - Running the script via the Home region. 

Reviewer 2: ```suggestion
. Running the Terraform script from the application (part 1) - Running the script via the Home region. 
```
ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#execute

. Preparing the {product-title} image for {oci-first-no-rt}. 

> . Executing the Terraform script (Part 2) - Running the script via the {oci-c3-short} region.

Reviewer 2: ```suggestion
. Running the Terraform script (Part 2) - Running the script via the {oci-c3-short} region.
```

> . Installing the cluster using the {ai-full} web console.

Reviewer 2: ```suggestion
. Installing the cluster by using the {ai-full} web console.
```
